### PR TITLE
Pass load balancer configs in nova metadata

### PR DESCRIPTION
**Pull request by cyli**
This addresses the first task in #968.

Punting on draining timeout because I don't know where to get it from yet.

### CODE REVIEW

File: otter/convergence/composition.py [hunk 1]
```diff
@@ -86,8 +93,35 @@ def get_desired_group_state(group_id, launch_config, desired):
...
+
+    for config in lb_args:
+        if config.get('type') != 'RackConnectV3':
+            sanitized = keyfilter(lambda k: k in ('type', 'port'), config)
+            # provide a default type
+            sanitized.setdefault('type', 'CloudLoadBalancer')
+
+            server_args = server_args.set_in(
+                ('server', 'metadata',
+                 'rax:autoscale:lb:{0}'.format(config['loadBalancerId'])),
```
Comment (lvh) [inline otter/convergence/composition.py]: While this is in fact the correct name of our product, there's a lot of "auto_scaling" going on in the codebase. Maybe we should have a plan for transitioning those over to a consistent naming scheme?

**Comment by lvh** [comment]
LGTM modulo that whole thing about consistent metadata key naming :)

File: otter/convergence/composition.py (hunk 1 above)
Comment (cyli) [inline otter/convergence/composition.py]: I think other teams actively use this particular key (in particularly the control panel, although I don't know who else) - we can add a new key, and tell everyone that one is used now, and to stop using this one, and kill this one off after a while.

Comment (cyli) [inline otter/convergence/composition.py]: https://github.com/rackerlabs/otter/issues/973

**Comment by cyli** [comment]
Oops, sorry, merge forward collapsed that convo.  Was there something else I should do with the metadata?  I filed https://github.com/rackerlabs/otter/issues/973

**Comment by lvh** [comment]
Nah, that's definitely a separate ticket.

**Pull request merged**